### PR TITLE
Aggregate, harvesting and navbar responsive

### PR DESCRIPTION
**Pull request by Peersyst**

### CODE REVIEW

File: src/views/pages/aggregate/Aggregate.vue [hunk 1]
```diff
@@ -1,13 +1,88 @@
...
+                        </span>
+                        <span class="transfer_name">asdfdsagfdga</span>
+                        <button class="remove_button">
+                            <img src="@/views/resources/img/icons/bin.svg" alt />
+                        </button>
+                    </div>
+                </div>
+                <div class="bottom-container">
+                    <button class="inverted-button pl-2 pr-2" style="cursor: pointer;">
+                        SEND AGGREGATE
```
Comment (rg911) [inline src/views/pages/aggregate/Aggregate.vue:70]: Language missing?

File: src/views/layout/PageLayout/PageLayout.vue [hunk 2]
```diff
@@ -13,45 +13,61 @@
...
+                        <AccountLinks
+                            v-if="isTestnet"
+                            :account="currentAccount"
+                            :link="faucetUrl"
+                            :icon="faucetIcon"
+                            :title="$t('accounts_links_faucet')"
+                            class="level-item navbar-item"
+                        />
+                        <!--<div class="level-item navbar-item debug-console-trigger" @click="hasDebugConsoleModal = true">
                     <span>&nbsp;{{ $t('top_window_console') }}</span>
```
Comment (rg911) [inline src/views/layout/PageLayout/PageLayout.vue:45]: Remove debug

File: src/language/en-US.json [hunk 3]
```diff
@@ -560,9 +561,9 @@
     "show_expired_namespaces": "Show expired namespaces",
     "show_wizard": "Show wizard",
     "sidebar_item_accounts": "Accounts",
+    "sidebar_item_harvesting": "Harvest",
```
Comment (rg911) [inline src/language/en-US.json:564]: - Language key-value missing in Jap and CHN
- Language JSON should be sorted by key

**Review by rg911 — CHANGES_REQUESTED** [review]
- A few small issues. nice one
- The main one is this frontend_restyle branch need to rebase the latest integration. There are a few commits are missing.

**Comment by rg911** [comment]
Just a suggestion can we add a bit of description when raising PRs? It would be easier for the reviewer. 

e.g. Tasks handled, function changed etc..

File: src/views/layout/PageLayout/PageLayout.vue (hunk 2 above)
Comment (Peersyst) [reply src/views/layout/PageLayout/PageLayout.vue]: Removed

File: src/language/en-US.json (hunk 3 above)
Comment (Peersyst) [reply src/language/en-US.json:564]: Fixed

File: src/views/pages/aggregate/Aggregate.vue (hunk 1 above)
Comment (Peersyst) [reply src/views/pages/aggregate/Aggregate.vue]: Translated

**Review by rg911 — APPROVED** [review]

**Pull request merged**